openedx/core/djangoapps/theming/helpers.py

```diff
@@ -298,7 +319,16 @@ def is_comprehensive_theming_enabled():
     Returns:
          (bool): True if comprehensive theming is enabled else False
     """
+    # Import is placed here to avoid circular import
+    from openedx.core.djangoapps.theming.microsite_transition_helpers import current_request_has_associated_site_theme
+
+    # We need to give priority to theming over microsites
+    if current_request_has_associated_site_theme():
+        return True
+
     # Disable theming for microsites
+    # We need to check microsite requests here becasue of we do not then
```

Comment:
I'd revise this comment to say "Microsite configurations take priority over the default site theme."